index.d.ts

```diff
@@ -332,6 +332,29 @@ export interface Options extends Omit<RequestInit, 'headers'> {
 	```
 	*/
 	onDownloadProgress?: (progress: DownloadProgress, chunk: Uint8Array) => void;
+
+	/**
+	User-defined `fetch` function.
+
+	Use-cases:
+	1. Use custom `fetch` implementations like [`isomorphic-unfetch`](https://www.npmjs.com/package/isomorphic-unfetch).
+	2. Use the `fetch` wrapper function provided by some frameworks that use server-side rendering (SSR).
+
+	@default fetch
+
+	@example
+	```
+	import ky from 'ky';
+	import fetch from 'isomorphic-unfetch';
+
+	(async () => {
+		const parsed = await ky('https://example.com', {
+			fetch: fetch
```

Comment:
Same as in the readme, you can use shorthand syntax here: `fetch` instead of `fetch: fetch`

Reply:
Had it like that at first but then I thought it could be kinda confusing for newbies... But sure I'll change it

Reply:
It's definitely something to get used to. Ultimately it simplifies things, though.